validphys2/src/validphys/process_options.py

```diff
@@ -306,8 +310,9 @@ def _dyncpt_xq2map(kin_info):
     "HQP_PTQ": HQP_PTQ,
     "HERAJET": HERAJET,
     "HERADIJET": dataclasses.replace(HERAJET, name="HERADIJET", description="DIS + jj production"),
-    "DY_W_ETA": DY_W_ETA,
-    "DY_NC_PT": DY_NC_PT,
+    "DY_Z_Y": dataclasses.replace(DY_2L, name="DY_Z_Y", description="DY Z -> ll (pseudo)rapidity"),
+    "DY_W_ETA": dataclasses.replace(DY_2L, name="DY_W_ETA", description="DY W -> l nu (pseudo)rapidity"),
+    "DY_NC_PT": dataclasses.replace(DY_2L_PT, name="DY_NC_PT", description="DY Z -> ll + j")
```

Comment:
Could you please check that this makes sense? I'm still using `DY_W_ETA` because the previously implemented datasets for DY_CC adopts `DY_W_ETA` as process type in the metadata.

Reply:
Ok, this is fine. I don't mind that much having too many different types at the dataset level, which I'd like to avoid is the explosion of functions that compute said datasets.

One thing though, for the PT I don't think `DY_2L_PT` is correct. Just use `DY_PT`.

```suggestion
    "DY_NC_PT": dataclasses.replace(DY_2L_PT, name="DY_NC_PT", description="Z + j")
```

If you want to be more specific, `Z (ll) +j` but with `Z -> ll +j ` it looks like it is the Z that decays into two leptons and a jet.

Reply:
Ok, I've modified that.